filters/code_mixing/README.md

```diff
@@ -0,0 +1,21 @@
+## Code-Mixing Filter
+
+## What type of a filter is this?
+
+This filter selects for code-mixed texts (i.e. texts using the multiple 
```

Comment:
nit: remove the "the". 

Also maybe elaborate on which languages this works with?

Reply:
Fixed typo. I also [mentioned the languages](https://github.com/GEM-benchmark/NL-Augmenter/blob/43aeab14f70877a57f3a7c806a7524c1027148a8/filters/code_mixing/README.md#what-are-the-limitations-of-this-filter) in the README. Is that alright?